modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -2719,6 +3006,8 @@ private void printHelp() {
         usage(i("Set baseline topology based on version:"), BASELINE, BaselineCommand.VERSION.text() + " topologyVersion", op(CMD_AUTO_CONFIRMATION));
         usage(i("Set baseline autoadjustment settings:"), BASELINE, BaselineCommand.AUTO_ADJUST.text(), "disable|enable timeout <timeoutValue>", op(CMD_AUTO_CONFIRMATION));
         usage(i("List or kill transactions:"), TX, getTxOptions());
+        usage(i("Print detailed information (topology and key lock ownership) about specific transaction:"),
+            TX, TX_INFO, or("<GridCacheVersion [topVer=..., order=..., nodeOrder=...]>", "<UUID>"));
```

Comment:
TX, TX_INFO, or("<TX identifier as GridCacheVersion [topVer=..., order=..., nodeOrder=...]>", "<TX identifier as UUID>"));

Also, I believe, that it's worth to mention that it could be found in logs.

Reply:
Fixed.